translations-app/handleAppTranslations.sh

```diff
@@ -49,7 +49,7 @@ do
   tx migrate
   git add --force .tx/config
   rm .tx/config_*
-  git commit -am "[tx-robot] Update transifex configuration" -s || true
+  git commit -am "Fix(l10n): 🛠️ Update Transifex configuration" -s || true
```

Comment:
not sure about `Fix` vs `fix`

Reply:
https://www.conventionalcommits.org/en/v1.0.0/#are-the-types-in-the-commit-title-uppercase-or-lowercase

Reply:
he units of information that make up Conventional Commits MUST NOT be treated as case sensitive by implementors, with the exception of BREAKING CHANGE which MUST be uppercase.